dev-embedded/ponyprog/files/ponyprog-3.0.0-install_doc.patch

```diff
@@ -0,0 +1,68 @@
+--- ponyprog-3.0.0/distribution/posix/CMakeLists.txt-	2017-09-30 03:35:21.000000000 +0300
++++ ponyprog-3.0.0/distribution/posix/CMakeLists.txt	2018-03-28 11:25:40.270001807 +0300
+@@ -5,6 +5,8 @@
+ SET(CPACK_INSTALL_PREFIX "/usr") 
+ SET(CPACK_PACKAGE_INSTALL_DIRECTORY "/usr")
+ 
++INCLUDE(GNUInstallDirs)
++
+ MESSAGE("CMAKE_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX}")
+ 
+ # SET( CPACK_PACKAGE_GROUP "${PONYPROG_CPACK_PACKAGE_GROUP}" )
+@@ -13,45 +15,40 @@
+ # SET( CPACK_PACKAGE_DEPENDS "${PONYPROG_CPACK_PACKAGE_DEPENDS}" )
+ 
+ INSTALL(PROGRAMS "${CURRENT_BUILD_DIR}/ponyprog" 
+-        DESTINATION "bin/" 
++        DESTINATION "${CMAKE_INSTALL_BINDIR}" 
+         COMPONENT "application" 
+-        PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ GROUP_EXECUTE GROUP_READ WORLD_EXECUTE WORLD_READ
+         )
+ 
+ INSTALL(DIRECTORY "${PROJECT_SOURCE_DIR}/icons/"    
+-        DESTINATION "share/icons/" 
++        DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/icons" 
+         COMPONENT "application" 
+-        DIRECTORY_PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ GROUP_EXECUTE GROUP_READ WORLD_EXECUTE WORLD_READ 
+         FILES_MATCHING PATTERN "*.png"
+         )
+ 
+ INSTALL(DIRECTORY "${PROJECT_SOURCE_DIR}/lang/" 
+-        DESTINATION "share/ponyprog/lang/" 
++        DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/ponyprog/lang" 
+         COMPONENT "lang files" 
+-        DIRECTORY_PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ GROUP_EXECUTE GROUP_READ WORLD_EXECUTE WORLD_READ 
+         FILES_MATCHING PATTERN "*.utf"
+         ) 
+ 
+ INSTALL(DIRECTORY "${PROJECT_SOURCE_DIR}/distribution/innosetup/" 
+-        DESTINATION "share/doc/ponyprog/" 
++        DESTINATION "${CMAKE_INSTALL_DOCDIR}" 
+         COMPONENT "help files" 
+-        DIRECTORY_PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ GROUP_EXECUTE GROUP_READ WORLD_EXECUTE WORLD_READ 
+         FILES_MATCHING PATTERN "*.jpg" PATTERN "*.html"
+         ) 
+ 
+ INSTALL(FILES "${PROJECT_SOURCE_DIR}/copyright" 
+-        DESTINATION "share/doc/ponyprog/" 
++        DESTINATION "${CMAKE_INSTALL_DOCDIR}" 
+         COMPONENT "application" 
+-        PERMISSIONS OWNER_WRITE OWNER_READ GROUP_READ WORLD_READ
+         ) 
+ # MESSAGE("PROJECT_SOURCE_DIR ${PROJECT_SOURCE_DIR}")
+ 
+-INSTALL(FILES "${PROJECT_SOURCE_DIR}/desktop/ponyprog.desktop"
+-        DESTINATION "share/applications/" 
+-        COMPONENT "application" 
+-        PERMISSIONS OWNER_WRITE OWNER_READ GROUP_READ WORLD_READ
+-        )
+-        
++#INSTALL(FILES "${PROJECT_SOURCE_DIR}/desktop/ponyprog.desktop"
```

Comment:
Why is this commented out?

Reply:
cause make_session_desktop used. is it ok?